app/Keycloak/ScopeConfig.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Keycloak;
+
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\IntegrationType;
+use Ramsey\Uuid\UuidInterface;
+
+final class ScopeConfig
+{
+    public function __construct(
+        public UuidInterface $searchApi,
```

Comment:
I was first confused about what this was, but it seems this is the scopeId of integration type SearchApi. Maybe make the name a bit more explicit?